packages/vetrina/src/Vetrina/Vetrina.purs

```diff
@@ -202,6 +204,9 @@ pollOrder setState state@{ logger } (Right order) = do
           nextPurchaseStep = case userAccountStatus of
             NewAccount      -> PurchaseSetPassword
             _               -> PurchaseCompleted userAccountStatus
+      product_id <- liftEffect $ LocalStorage.getItem "product_id" --analytics
+      product_price <- liftEffect $ LocalStorage.getItem "product_price" --analytics
```

Comment:
Do we want to send a warning to sentry if `product_id` and `product_price` are not found in the local storage? Do we care?

BTW let's not mix `snake_case` with `camelCase`, so rename these to `productId` and `productPrice` (however it's fine to have `"product_price"` in the local storage if you want)